docs/tasks/configuring-a-platform.md

```diff
@@ -311,6 +312,48 @@ kubectl patch deployment nuclio-dashboard \
   ]'
 ```
 
+<a id="base-images"></a>
+### Base Images (`baseImages`)
+
+The `baseImages` configuration allows you to override the default base images used for building function processor images on a per-runtime basis.
+This is useful when you need to use custom base images, such as in air-gapped environments or when using private registries.
+
+The configuration is a map where:
+- **Key**: Runtime name (e.g., `python`, `nodejs`) or runtime name with version (e.g., `python:3.12`, `nodejs:20`)
+- **Value**: The base image to use for that runtime
+
+When both a version-specific key (e.g., `golang:1.21`) and a runtime-name-only key (e.g., `golang`) are present, the version-specific key takes precedence.
```

Comment:
it makes an impression that we support golang versioning the same way we do with python, which is not true. Let's change example to use python